version.go

```diff
@@ -27,7 +27,7 @@ package gocql
 import "runtime/debug"
 
 const (
-	mainModule = "github.com/gocql/gocql"
+	defaultDriverName = "github.com/apache/cassandra-gocql-driver"
```

Comment:
Allows us to contain the default value in a distinct location (i.e. not embedded within the code below) with a _much_ more descriptive name.